core/server/admin.js

```diff
@@ -21,7 +21,7 @@ var zlib = require('zlib');
 
 var PORT = 4001;
 var DEBUG = process.env.NODE_ENV === "development" ? true : false;
```

Comment:
no need for `? true : false`, `process.env.NODE_ENV === "development"` is already a boolean